Documentation/cloud-config.md

```diff
@@ -41,20 +41,49 @@ CoreOS tries to conform to each platform's native method to provide user data. E
 
 #### etcd
 
-The `coreos.etcd.*` parameters will be translated to a partial systemd unit acting as an etcd configuration file.
+The `coreos.etcd.*` and `coreos.etcd2.*` parameters will be translated to a partial systemd unit acting as an etcd configuration file.
```

Comment:
We should make it the "etcd2" section and create a second section that is "etcd (deprecated, see etcd2)". /cc @robszumski